pkg/shell/machines/machines.js

```diff
@@ -220,15 +220,15 @@ function Machines() {
         }
 
         // FIXME: investigate re-using the proxy from Loader (runs in different frame/scope)
-        const bridge = cockpit.dbus(null, { bus: "internal", superuser: "try" });
+        const bridge = cockpit.dbus(null, { bus: "internal", superuser: "require" });
         const mod =
             bridge.call("/machines", "cockpit.Machines", "Update", ["99-webui.json", host, values_variant])
-                    .catch(function(error) {
-                        console.error("failed to call cockpit.Machines.Update(): ", error);
+                    .catch(error => {
+                        // avoid make noise when we are not superuser
+                        if (error.problem !== "access-denied")
+                            console.error("failed to call cockpit.Machines.Update(): ", JSON.stringify(error));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19506-20231019-123114-02c8ffd1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Right, it's a "Should Not Happen™" now.